server/mailet/rate-limiter-redis/src/main/scala/org/apache/james/rate/limiter/redis/RedisRateLimiter.scala

```diff
@@ -46,10 +44,6 @@ class RedisRateLimiterModule() extends AbstractModule {
       .to(classOf[RedisRateLimiterFactory])
 
     bind(classOf[RedisRateLimiterFactory]).in(Scopes.SINGLETON)
-
-    Multibinder.newSetBinder(binder(), classOf[HealthCheck])
-      .addBinding()
-      .to(classOf[RedisHealthCheck])
```

Comment:
Note that our TMail is using this `RedisRateLimiterModule`, we need to bind `RedisHealthCheck` explicitly, otherwise, we lose this on the TMail side.